stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamDimens.kt

```diff
@@ -177,7 +192,15 @@ constructor(
      * users.
      * @param attachmentsContentImageMaxHeight The maximum height an image attachment will expand to while automatically
      *  re-sizing itself in order to obey its aspect ratio.
+     * @param attachmentsContentVideoMaxHeight The maximum height video attachment will expand to while automatically
+     *  re-sizing itself in order to obey its aspect ratio.
      */
+    @Deprecated(
```

Comment:
Deprecating this constructor is unfortunate but a lot of work was done + new work will be done with giphy optimization so it was unavoidable.

Ultimately though it's just a constructor so users shouldn't have much issue switching.